
----------------------train start--------------------------
  0%|                                                    | 0/25 [00:00<?, ?it/s]/home/kai/.conda/envs/vindr/lib/python3.12/site-packages/torch/nn/modules/conv.py:456: UserWarning: Applied workaround for CuDNN issue, install nvrtc.so (Triggered internally at /opt/conda/conda-bld/pytorch_1712609048481/work/aten/src/ATen/native/cudnn/Conv_v8.cpp:84.)
  return F.conv2d(input, weight, bias, self.stride,





 92%|███████████████████████████████████████▌   | 23/25 [00:11<00:00,  2.04it/s]
100%|███████████████████████████████████████████| 25/25 [00:12<00:00,  2.01it/s]
 29%|████████████▊                                | 2/7 [00:00<00:02,  2.02it/s]

100%|█████████████████████████████████████████████| 7/7 [00:03<00:00,  2.26it/s]
Traceback (most recent call last):
  File "/mnt/data/kai/VinDr_Code/Mini_VinDr_CXR_Dataset/pytorch_model_training/train.py", line 176, in <module>
    if early_stopper.early_stop(validation_loss):
    ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ValidationLossEarlyStopping' object has no attribute 'early_stop'